codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -718,6 +718,13 @@ void onHttp2Frame(ChannelHandlerContext ctx, Http2Frame frame) {
         ctx.fireChannelRead(frame);
     }
 
+    /**
+     * Create a Http2UnknownFrame.
+     * */
+    protected Http2UnknownFrame newHttp2UnknownFrame(byte frameType, int streamId, Http2Flags flags, ByteBuf payload) {
```

Comment:
@He-Pin just to double check... its fine that this returns `Http2UnknownFrame` and not `Http2Frame` correct ?

Reply:
It's fine, as I can extends the Http2UnkownFrame too, but returns a Http2Frame would be more user friendly

Reply:
Actually it needs to be `Http2FrameStream`. Changing